src/app/components/shared/datetime/datetime/datetime.component.spec.ts

```diff
@@ -0,0 +1,156 @@
+import { Spectator, createComponentFactory } from "@ngneat/spectator";
+import { SharedModule } from "@shared/shared.module";
+import { MockBawApiModule } from "@baw-api/baw-apiMock.module";
+import { DateTime, Settings } from "luxon";
+import { assertTooltip } from "@test/helpers/html";
+import { DatetimeComponent } from "./datetime.component";
+
+describe("DatetimeComponent", () => {
+  let spectator: Spectator<DatetimeComponent>;
+
+  const createComponent = createComponentFactory({
+    component: DatetimeComponent,
+    imports: [SharedModule, MockBawApiModule],
+  });
+
+  function setup(): void {
+    // to ensure that tests pass for all timezones, we set the default timezone to Australia/Perth
+    // this mocks the test runners timezone to be Australia/Perth so that they will assert correctly in all timezones
+    Settings.defaultZone = "Australia/Perth";
+    Date.prototype.getTimezoneOffset = () => {
+      return 8 * 60; // +08:00 hours in minutes (8 * 60)
+    };
+
+    // utc datetime should be localized to "Australia/Perth"
+    let fakeDateTime = DateTime.fromISO("2020-01-01T12:10:11.000Z");
+    fakeDateTime = fakeDateTime.toUTC();
```

Comment:
why are we setting the components value here, when you override it in each test?